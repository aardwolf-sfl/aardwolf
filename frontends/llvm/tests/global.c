int GLOBAL = 0;

int get_global() {
    GLOBAL++;
    return GLOBAL;
}
