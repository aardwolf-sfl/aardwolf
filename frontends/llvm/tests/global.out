
function: get_global

#1 -> #2  ::  defs: %1 / uses: %1 [@1, 4, 11]
#2 ->   ::  defs:  / uses: %1 [@1, 5, 5]

@1 = tests/global.c
